src/styles/main.scss

```diff
@@ -0,0 +1,72 @@
+$watch-size: 600px;
+$watch-radius: $watch-size/2;
+$watch-minute-height: $watch-radius / 2;
+$watch-second-height: $watch-radius;
+$watch-minute-width: 6px;
+$watch-second-width: $watch-minute-width*2;
+$pin-size: 40px;
+$stop-watch-time: 60s;
+
+@mixin circle ($size) {
+  height: $size;
+  width: $size;
+  border-radius: 50%;
+}
+
+body {
+  margin: 0;
+}
+
+.watch {
+  position: relative;
+  @include circle($watch-size);
+  background-color: rgb(156, 171, 184);
+}
+
+.page__watch {
+  margin: 0 auto;
+}
+
+.watch__hand {
+  position: absolute;
+
+}
+
+.watch__hand--min {
+  height: $watch-minute-height;
+  width: $watch-minute-width;
+  top: $watch-radius - $watch-minute-height;
+  left: $watch-radius - $watch-minute-width;
+  background-color: blue;
+  transform-origin: bottom right;
+  animation: rotate $stop-watch-time*60 linear infinite;
+}
+
+.watch__hand--sec {
+  height: $watch-radius;
+  width: $watch-second-width;
+  background-color: red;
+  top: 0;
+  left: $watch-radius - $watch-second-width;
```

Comment:
If you want to position your handle in the center you need to subtract half of its thickness, not the full thickness. So   left: `$watch-radius - ($watch-second-width / 2);` will look better.